### DISCUSSION TITLE
Speechly React library support for Next.js and server-side rendering

### DISCUSSION THREAD

**Discussion by arzga** [speechly/speechly#116] [Announcements]
Just published v0.0.23 of [react-client](https://www.npmjs.com/package/@speechly/react-client) and v2.3.0 of [react-ui](https://www.npmjs.com/package/@speechly/react-ui) that play nice with [Next.js](https://nextjs.org/)'s server-side page pre-rendering.

This is achieved by doing Speechly initialization that depends on browser APIs on DOM mount instead of component creation.

Please let us know how you think!

**Comment by arzga** [comment]
There is now a `/examples/next-js-example/` in this monorepository. It's a two-page Next.js site bootstrapped with `yarn create next-app --typescript` with Speechly UI components on them.

There's instructions on how to use the Rush monorepo on https://github.com/speechly/speechly#how-to-use-this-rush-monorepository

**Comment by khalidfarooq** [comment]
Hi Arzga, that repo is no longer there. Is there any alternative?
I want to do real time voice to text transcription in my nextjs web app
